.github/workflows/daily_ete_test.yml

```diff
@@ -236,13 +242,6 @@ jobs:
         run: |
           pytest autotest/tools/restful/test_restful_chat_hf_pytorch.py -m 'gpu_num_1 and not pr_test' -n 8 --alluredir=allure-results || true
           pytest autotest/tools/restful/test_restful_chat_hf_pytorch.py -m 'gpu_num_2 and not pr_test' -n 4 --alluredir=allure-results
-      - name: Test lmdeploy - interface pipeline turbomind case
```

Comment:
What benefits can be gained by moving this snippet to the `test_pipeline` workflow?

Reply:
> What benefits can be gained by moving this snippet to the `test_pipeline` workflow?

test tools want to cover more models's basic function.
test restful、test pipeline  want to cover more functions of this two interface by using specific models. 

Moving pipeline interface testcase out make the testcase's motivation more clear. And also can help me to manage and rerun failed functions separately. 

For example, Some model may leads test tools unstable. I can run interface testcase separately.